crates/hir_ty/src/tests/coercion.rs

```diff
@@ -255,6 +263,9 @@ fn coerce_fn_item_to_fn_ptr() {
 fn foo(x: u32) -> isize { 1 }
 fn test() {
     let f: fn(u32) -> isize = foo;
+                           // ^^^ adjustments: Pointer(ReifyFnPointer)
+    let f: unsafe fn(u32) -> isize = foo;
+                                  // ^^^ adjustments: Pointer(ReifyFnPointer)
```

Comment:
And this adjustment should have a `Pointer(UnsafeFnPointer)` but it does not for some reason

Reply:
Ah we don't save unsafety of fn pointers in the signature yet